### PR TITLE
Add a configuration item to display / hide dot files.

### PR DESCRIPTION
**Pull request by dkam**
How does this look?

### CODE REVIEW

**Comment by rob-murray** [comment]
Wow. Lots of PRs :tada: 

Looks like @houndci has gone nuts too.

Looks good to me.

File: bin/ferver [hunk 1]
```diff
@@ -9,6 +9,10 @@ OptionParser.new do |opts|
   opts.banner = 'Ferver: A simple web app to serve files over HTTP. Version: ' + Ferver::VERSION
   opts.separator ''
 
+  opts.on('-i', '--invisible', 'Serve invisible files') do |i|
```
Comment (rob-murray) [inline bin/ferver]: Maybe `hidden` instead as its a universal term for dot files?

Comment (dkam) [inline bin/ferver]: Yes - that's a better name.  What's the short switch though? Can't be -h - how about -a for serve all files  ( as used by list command  ls -a ) ?

Comment (rob-murray) [inline bin/ferver]: Yeah, thats better - go for `-a` and `--all`

Comment (dkam) [inline bin/ferver]: Cool - all done!

**Pull request merged**